features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/tracking/MixedViewTrackingStrategy.kt

```diff
@@ -50,6 +48,8 @@ class MixedViewTrackingStrategy internal constructor(
         )
     )
 
+    // region ActivityLifecycleTrackingStrategy
+
     override fun register(sdkCore: SdkCore, context: Context) {
```

Comment:
I think this was brought with the sdk v2 as before we did not register the strategies.